Xcodes/Frontend/Common/ObservingProgressIndicator.swift

```diff
@@ -42,6 +42,7 @@ public struct ObservingProgressIndicator: View {
             isIndeterminate: progress.progress.isIndeterminate,
             style: style
         )
+        .help("Dowloading: \(Int((progress.progress.fractionCompleted * 100)))% complete")
```

Comment:
Should we fix the typo here?
```suggestion
        .help("Downloading: \(Int((progress.progress.fractionCompleted * 100)))% complete")
```